### PR TITLE
Mfe source update

### PR DESCRIPTION
**Pull request by maryarbab**
Is this a new addition or an update to an existing factor?

Update

What category or categories does it affect?

Multiple categories  

What?

Updates to the year, source link, value, and description fields.

Why?

To link to the most recent Mfe source.

### CODE REVIEW

**Review by greenercode — APPROVED** [review]

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -371,7 +371,7 @@ Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kg-CO2e/kWh,0.338
...
 Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kg-CO2e/kWh,0.0013,,UNFCCC,2018,CA-QC,2021/07/30,Emission intensity of grid mix as reported. Published in the UN Framework Convention on Climate Change 2020. Includes generation factor.,https://data.ec.gc.ca/data/substances/monitor/canada-s-official-greenhouse-gas-inventory/Emission_Factors.pdf,electricity_generation
-Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kg-CO2e/kWh,0.0977,,MfE,2019,NZ,2021/09/10,Emission intensity of grid mix as reported. Published by the Ministry for the Environment of New Zealand. Based on the 2018 national inventory which is based on 2016 data. Includes production fuel mix factor. Includes generation factor.,https://www.mfe.govt.nz/node/18670/,electricity_generation
+Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kg-CO2e/kWh,0.101,,MfE,2020,NZ,2021/09/10,Emission intensity purchased grid-average electricity. Published by the Ministry for the Environment of New Zealand. Available in MfE Guidance for Voluntary GHG Reporting - 2020 Detailed Giude. ,https://www.mfe.govt.nz/node/18670/,electricity_generation
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv]: I don't really understand the sentence "Emission intensity purchased grid-average electricity" - should this be "Emission intensity of electricity purchased from the grid"?

Comment (patstirling) [reply OpenEmissionFactorsDB.csv]: Suggest replacing all "Available in" with "Retrieved from"

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -499,14 +499,14 @@ Energy,Electricity,electricity-energy_source_onshore_wind_turbine,Onshore wind t
...
-Energy,Fuel,fuel_type_avtur-fuel_use_aviation,Aviation turbine fuel (avtur),kg-CO2e/L,2.62632,,MfE,2020,NZ,2021/09/10,Emission intensity of aviation fuel (kerosene)/Jet A1. Published by the Ministry for the Environment of New Zealand in 2020.,https://environment.govt.nz/publications/measuring-emissions-detailed-guide-2020/,fuel_combustion
+Energy,Fuel,fuel_type_avtur-fuel_use_aviation,Aviation turbine fuel (avtur),kg-CO2e/L,2.62632,,MfE,2020,NZ,2021/09/10,Emission intensity of aviation gasoline. Published by the Ministry for the Environment of New Zealand in 2020. Available in 2020 Emission Factors Flat File. ,https://environment.govt.nz/publications/measuring-emissions-detailed-guide-2020/,fuel_combustion
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv]: There appears to be a copy and paste error in this line and many others (they all have "aviation gasoline" - please check!

File: OpenEmissionFactorsDB.csv [hunk 3]
```diff
@@ -684,7 +684,7 @@ Health and Social Care,Health Care,health_care-type_healthcare_practitioners_exc
...
+Health and Social Care,Health Care,health_care-type_medical_services,Medical services,kg-CO2e/GBP,0.297,,BEIS,2018,GB,2021/11/10,Emission intensity of supply chain in GBP spend on: NHS medical/optical/dental and medical auxiliary services/ private medical optical dental and auxiliary services/other health services/ in-patient hospital services. In actual prices in £s and including VAT. Provided by BEIS/DEFRA in the official report on GHG Emissions of the UK between 1996-2018. Where source emission factors are identical across a variety of activities they have been grouped to avoid confusion - details are documented in the OEFDB data guidance.,https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1000336/Consumtion_emissions_March_21_final_accessible_rev.ods,unknown
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv]: this change has been made in other PRs - remove to avoid conflicts

File: OpenEmissionFactorsDB.csv [hunk 4]
```diff
@@ -1095,8 +1095,8 @@ Materials,Mined Materials,mined_materials-type_copper_nickel_lead_zinc,Copper/ni
...
-Materials,Mining ,mining-type_well_drilling,Well drilling,kg-CO2e/USD,0.543,,EPA,2020,US,2021/10/12,Emission intensity of supply chain (with margins i.e. cradle to shelf) in US dollars spend on: well drilling. This factor is representative of the described commodity (equivalent to a goods or services category) and was calculated from 2016 data applying IPCC 4th Assessment Report conversion factors.,https://edg.epa.gov/metadata/catalog/search/resource/details.page?uuid=https://doi.org/10.23719/1517796,cradle_to_shelf
+Materials,Mining,mining-type_other_support_activities_for_mining,Other support activities for mining,kg-CO2e/USD,0.613,,EPA,2020,US,2021/10/12,Emission intensity of supply chain (with margins i.e. cradle to shelf) in US dollars spend on: other support activities for mining. This factor is representative of the described commodity (equivalent to a goods or services category) and was calculated from 2016 data applying IPCC 4th Assessment Report conversion factors.,https://edg.epa.gov/metadata/catalog/search/resource/details.page?uuid=https://doi.org/10.23719/1517796,cradle_to_shelf
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv]: remove to avoid conflicts

File: OpenEmissionFactorsDB.csv [hunk 5]
```diff
@@ -1095,8 +1095,8 @@ Materials,Mined Materials,mined_materials-type_copper_nickel_lead_zinc,Copper/ni
...
+Materials,Mining,mining-type_well_drilling,Well drilling,kg-CO2e/USD,0.543,,EPA,2020,US,2021/10/12,Emission intensity of supply chain (with margins i.e. cradle to shelf) in US dollars spend on: well drilling. This factor is representative of the described commodity (equivalent to a goods or services category) and was calculated from 2016 data applying IPCC 4th Assessment Report conversion factors.,https://edg.epa.gov/metadata/catalog/search/resource/details.page?uuid=https://doi.org/10.23719/1517796,cradle_to_shelf
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv]: remove to avoid conflicts

File: OpenEmissionFactorsDB.csv [hunk 6]
```diff
@@ -2093,4 +2093,4 @@ Waste,Plastic Waste,waste_type_pvc_including_forming-disposal_method_open_loop,P
...
-Water,Water Treatment,water-treatment,Water treatment,kg-CO2e/m3,0.272,,BEIS,2021,GB,2021/07/23,Emission Intensity of water treatment in the UK in 2021. Based on emissions associated with offices/transport/treatment of wastewater. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,upstream
+Water,Water Treatment,water-treatment,Water treatment,kg-CO2e/m3,0.272,,BEIS,2021,GB,2021/07/23,Emission Intensity of water treatment in the UK in 2021. Based on emissions associated with offices/transport/treatment of wastewater. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,upstreams
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv]: remove this change - looks like a type

**Review by patstirling — CHANGES_REQUESTED** [review]

File: OpenEmissionFactorsDB.csv [hunk 7]
```diff
@@ -577,10 +577,10 @@ Energy,Fuel,fuel_type_petrol-fuel_use_na,Petrol,kg-CO2e/L,2.33969,,BEIS,2021,GB,
...
 Energy,Fuel,fuel_type_petrol_biofuel_blend-fuel_use_na,Petrol (biofuel blend),kg-CO2e/t,2947.62,,BEIS,2021,GB,2021/11/17,Emission intensity for petrol (biofuel blend). The fuels conversion factors should be used for primary fuel sources combusted at a site or in an asset owned or controlled by the reporting organisation. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,fuel_combustion
-Energy,Fuel,fuel_type_petrol-fuel_use_transport,Petrol (transport),kg-CO2e/L,2.45323824,,MfE,2020,NZ,2021/09/10,Emission intensity of regular petrol. Published by the Ministry for the Environment of New Zealand in 2020.,https://environment.govt.nz/publications/measuring-emissions-detailed-guide-2020/,fuel_combustion
+Energy,Fuel,fuel_type_petrol-fuel_use_transport,Petrol (transport),kg-CO2e/L,2.45323824,,MfE,2020,NZ,2021/09/10,Emission intensity of aviation gasoline. Published by the Ministry for the Environment of New Zealand in 2020. Available in 2020 Emission Factors Flat File. ,https://environment.govt.nz/publications/measuring-emissions-detailed-guide-2020/,fuel_combustion
```
Comment (AnastasiaLobanova) [inline OpenEmissionFactorsDB.csv]: Is this correct? Should the id reflect it is the aviation gasoline? I thought avgas has strict requirements on its quality, higher than average petrol? Happy to be corrected :)

File: OpenEmissionFactorsDB.csv [hunk 8]
```diff
@@ -1331,7 +1331,7 @@ Restaurants and Accommodation,Food and Beverage Services,consumer_services-type_
...
-Transport,Air Transport,freight_flight-route_type_domestic-distance_na-weight_na-contrails_included,Domestic air freight,kg-CO2e/tkm,4.76719,,MfE,2020,NZ,2021/09/10,Emission intensity of domestic freight flight in New Zealand including emissions from combustion and contrails but not upstream fuel emissions. Published by the Ministry for the Environment of New Zealand in 2020.,https://environment.govt.nz/publications/measuring-emissions-detailed-guide-2020/,fuel_combustion
+Transport,Air Transport,freight_flight-route_type_domestic-distance_na-weight_na-contrails_included,Domestic air freight,kg-CO2e/tkm,4.76719,,MfE,2020,NZ,2021/09/10,Emission intensity of aviation gasoline. Published by the Ministry for the Environment of New Zealand in 2020. Available in 2020 Emission Factors Flat File. ,https://environment.govt.nz/publications/measuring-emissions-detailed-guide-2020/,fuel_combustion
```
Comment (AnastasiaLobanova) [inline OpenEmissionFactorsDB.csv]: Looks like a replacement error?

File: OpenEmissionFactorsDB.csv [hunk 9]
```diff
@@ -1343,50 +1343,50 @@ Transport,Air Transport,freight_flight-route_type_domestic-distance_gt_500km_lt_
...
-Transport,Air Transport,freight_flight-route_type_international-distance_long_haul-weight_na-contrails_included,International air freight long haul,kg-CO2e/tkm,1.13382,,MfE,2020,NZ,2021/09/10,Emission intensity of international long-haul freight flight to or from New Zealand including emissions from combustion and contrails but not upstream fuel emissions. . Published by the Ministry for the Environment of New Zealand in 2020.,https://environment.govt.nz/publications/measuring-emissions-detailed-guide-2020/,fuel_combustion
+Transport,Air Transport,freight_flight-route_type_international-distance_long_haul-weight_na-contrails_included,International air freight long haul,kg-CO2e/tkm,1.13382,,MfE,2020,NZ,2021/09/10,Emission intensity of aviation gasoline. Published by the Ministry for the Environment of New Zealand in 2020. Available in 2020 Emission Factors Flat File. ,https://environment.govt.nz/publications/measuring-emissions-detailed-guide-2020/,fuel_combustion
```
Comment (AnastasiaLobanova) [inline OpenEmissionFactorsDB.csv]: same here - is it a typo?

**Review by AnastasiaLobanova — CHANGES_REQUESTED** [review]
I think there is a problem with replacements - "aviation gasoline" appears in some places it should not be

**Pull request closed without merging**